code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -613,13 +649,27 @@
 					message = "buzzes."
 				playsound(src.loc, 'sound/machines/buzz-sigh.ogg', 50, 0)
 				m_type = 1
+		
+		if("vomit")
+			if (isipc(src))
+				src << "<span class='warning'>You are unable to vomit.</span>"
+				return
+			if (emotecooldown > world.time) 
+				src << "<span class='warning'>You are unable to vomit so soon.</span>"
+				emotecooldown = emotecooldown + 4 SECONDS
+				return
+			message = "vomits!"
+			vomit()
+			emotecooldown = world.time + 10 SECONDS
+			return
+
 
 		if ("help")
 			src << {"blink, blink_r, blush, bow-(none)/mob, burp, choke, chuckle, clap, collapse, cough,
 cry, custom, deathgasp, drool, eyebrow, frown, gasp, giggle, groan, grumble, handshake, hug-(none)/mob, glare-(none)/mob,
 grin, laugh, look-(none)/mob, moan, mumble, nod, pale, point-atom, raise, salute, shake, shiver, shrug,
 sigh, signal-#1-10, smile, sneeze, sniff, snore, stare-(none)/mob, tremble, twitch, twitch_s, whimper,
-wink, yawn, swish, sway/wag, fastsway/qwag, stopsway/swag, beep, ping, buzz"}
+wink, yawn, swish, sway/wag, fastsway/qwag, stopsway/swag, beep, ping, buzz, slap, snap"}

```

Comment:
Either document vomit, or remove it.

Reply:
There's other various emotes that arent documented, including *dap, though I will admit that I just forgot to add it to the list.